src/inputs/pcap/PcapInputStream.cpp

```diff
@@ -172,7 +172,7 @@ void PcapInputStream::start()
                 currInterface = currInterface->next;
```

Comment:
so now that we have clone(), we want to remove this whole block and avoid calling libpcap functions directly. all we have to do now is set _pcapDevice (from block on 144) to pcapDevice->clone()